next-sitemap.config.js

```diff
@@ -28,29 +28,40 @@ module.exports = {
 		'sitemap.xml',
 		'/server-sitemap.xml',
 		'/boundary/docs*',
+		'/boundary/tutorials*',
 		'/consul/docs*',
 		'/consul/commands*',
 		'/consul/api-docs*',
+		'/consul/tutorials*',
 		'/nomad/docs*',
 		'/nomad/plugins*',
 		'/nomad/intro*',
 		'/nomad/tools*',
```

Comment:
So if a new docs sub path is added, we'll need to remember to add it here? I could see that slipping through the cracks. Is there a way we could compose this from the docs base paths in the data files? Could be a future thought, not or this PR.